akka-client/src/main/scala/com/improving/app/gateway/api/GatewayClient.scala

```diff
@@ -0,0 +1,21 @@
+package com.improving.app.gateway.api
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import com.typesafe.config.ConfigFactory
+
+object GatewayClient {
+  def initiateClient(): GatewayServiceClient = {
+    // Boot akka
+    implicit val sys =
+      ActorSystem("UIGatewayClient", ConfigFactory.load("application.conf"))
```

Comment:
Since we drpped "UI" in other places, perhaps it would be consistent to call the Actor System "GatewayClient" ?

Reply:
Good catch, thought I had caught them all. Will do a global search just in case now